--- fvwm-2.6.6.orig/modules/FvwmScroll/GrabWindow.c
+++ fvwm-2.6.6/modules/FvwmScroll/GrabWindow.c
@@ -80,7 +80,7 @@ extern char *BackColor;
 		     ButtonPressMask | ButtonMotionMask | FocusChangeMask)

 Atom wm_del_win;
-Atom _XA_WM_PROTOCOLS;
+extern Atom _XA_WM_PROTOCOLS;
 Atom _XA_WM_COLORMAP_WINDOWS;

 /*
--- fvwm-2.6.6.orig/modules/FvwmWharf/FvwmWharf.c
+++ fvwm-2.6.6/modules/FvwmWharf/FvwmWharf.c
@@ -177,7 +177,7 @@ char *imagePath = NULL;
 char *imagePath = NULL;

 static Atom _XA_WM_DEL_WIN;
-Atom _XA_WM_PROTOCOLS;
+extern Atom _XA_WM_PROTOCOLS;
 Atom _XA_WM_NAME;
 #ifdef ENABLE_DND
 Atom DndProtocol;
--- fvwm-2.6.6.orig/modules/FvwmIconBox/FvwmIconBox.c
+++ fvwm-2.6.6/modules/FvwmIconBox/FvwmIconBox.c
@@ -188,7 +188,7 @@ char *FvwmDefaultIcon = NULL;
 char *FvwmDefaultIcon = NULL;

 static Atom wm_del_win;
-Atom _XA_WM_PROTOCOLS;
+extern Atom _XA_WM_PROTOCOLS;
 Atom _XA_WM_NAME;

 int ready = 0;
