diff -urN fvwm-2.5.12.vanilla/configure.in fvwm-2.5.12/configure.in
--- fvwm-2.5.12.vanilla/configure.in	2005-02-21 21:46:48.924687776 +0100
+++ fvwm-2.5.12/configure.in	2005-02-21 21:48:37.672155648 +0100
@@ -794,7 +794,7 @@
 AC_SUBST(FVWM_PERLLIB)
 
 dnl ******* gettext
-ALL_LINGUAS="ar de fr sv_SE zh_CN"
+ALL_LINGUAS="ar de fr sv zh"
 FVWM_DOMAIN="fvwm"
 FVWMTASKBAR_DOMAIN="FvwmTaskBar"
 FVWMSCRIPT_DOMAIN="FvwmScript"
