--- fvwm-2.7.0.orig/modules/FvwmPager/x_pager.c
+++ fvwm-2.7.0/modules/FvwmPager/x_pager.c
@@ -1143,6 +1143,29 @@
     else if((Event->xbutton.button == 1)||
 	    (Event->xbutton.button == 2))
     {
+		if( (Event->xbutton.button == 1) &&
+		  (Event->xbutton.subwindow != None) )
+		{
+		  PagerWindow *t;
+
+		  t = Start;
+		  while ((t != NULL)&&(t->PagerView != Event->xbutton.subwindow))
+		    t= t->next;
+
+		  if(t==NULL)
+		  {
+		    t = Start;
+		    while ((t != NULL)&&(t->IconView != Event->xbutton.subwindow))
+		      t= t->next;
+		  }
+
+		  if( t )
+		  {
+		    SendText(fd, "Silent Raise", t->w);
+		    SendText(fd, "Silent Focus", t->w);
+		  }
+		}
+
       for(i=0;i<ndesks;i++)
       {
 	if(Event->xany.window == Desks[i].w)
