diff -uNr fvwm-snap-20031025-org/modules/FvwmIconMan/FvwmIconMan.1 fvwm-snap-20031025/modules/FvwmIconMan/FvwmIconMan.1
--- fvwm-snap-20031025-org/modules/FvwmIconMan/FvwmIconMan.1	Sat Apr 19 13:30:03 2003
+++ fvwm-snap-20031025/modules/FvwmIconMan/FvwmIconMan.1	Sat Oct 25 17:00:44 2003
@@ -361,9 +361,11 @@
 .IP "*FvwmIconMan: [id] ShowTransient \fIboolean\fP"
 Show transient windows in the list (default false).
 
-.IP "*FvwmIconMan: [id] ShowOnlyIcons \fIboolean\fP"
-Only iconified windows are shown if \fIboolean\fP is true.
-
+.IP "*FvwmIconMan: [id] ShowOnlyIcons \fIvalue\fP"
+If set to "false" both normal and iconified windows are shown. If "true" then
+only iconified windows will get displayed, and "never" means that no iconified
+windows will be shown.
+ 
 .SH ACTIONS
 Actions are commands which may be bound to an event of the type: a key press, a
 mouse click, or the mouse entering a window manager button - denoted by the
diff -uNr fvwm-snap-20031025-org/modules/FvwmIconMan/FvwmIconMan.h fvwm-snap-20031025/modules/FvwmIconMan/FvwmIconMan.h
--- fvwm-snap-20031025-org/modules/FvwmIconMan/FvwmIconMan.h	Tue Oct 21 07:52:36 2003
+++ fvwm-snap-20031025/modules/FvwmIconMan/FvwmIconMan.h	Sat Oct 25 17:01:33 2003
@@ -300,7 +300,7 @@
 	WeightedSort *weighted_sorts;
 	int weighted_sorts_len, weighted_sorts_size;
 	char *AnimCommand;
-	Uchar showonlyiconic;
+	int showonlyiconic;
 	Uchar showtransient;
 	rectangle managed_g;    /* dimensions of managed screen portion */
 
diff -uNr fvwm-snap-20031025-org/modules/FvwmIconMan/readconfig.c fvwm-snap-20031025/modules/FvwmIconMan/readconfig.c
--- fvwm-snap-20031025-org/modules/FvwmIconMan/readconfig.c	Sat Jul  5 13:30:03 2003
+++ fvwm-snap-20031025/modules/FvwmIconMan/readconfig.c	Sat Oct 25 17:16:56 2003
@@ -1497,11 +1497,14 @@
 	 ConsoleMessage("Need argument to showonlyicons\n");
 	 continue;
        }
-       if (!strcasecmp(p, "true")) {
+       if (!strcasecmp (p, "false")) {
+	 i = 0;
+       }
+       else if (!strcasecmp (p, "true")) {
 	 i = 1;
        }
-       else if (!strcasecmp(p, "false")) {
-	 i = 0;
+       else if (!strcasecmp (p, "never")) {
+	 i = 2;
        }
        else {
 	 ConsoleMessage("Bad line: %s\n", current_line);
diff -uNr fvwm-snap-20031025-org/modules/FvwmIconMan/xmanager.c fvwm-snap-20031025/modules/FvwmIconMan/xmanager.c
--- fvwm-snap-20031025-org/modules/FvwmIconMan/xmanager.c	Tue Oct 21 07:52:36 2003
+++ fvwm-snap-20031025/modules/FvwmIconMan/xmanager.c	Sat Oct 25 17:30:57 2003
@@ -1476,11 +1476,15 @@
 
   if (win->manager && win->complete) {
     is_state_selected =
-	    ((!win->manager->showonlyiconic || win->iconified) &&
-	     (win->manager->showtransient || !IS_TRANSIENT(win)));
+	    (((!win->iconified&&(win->manager->showonlyiconic==2))||
+	    (win->manager->showonlyiconic==0)||
+	    ((win->iconified&&(win->manager->showonlyiconic==1))
+		))&&
+		 (win->manager->showtransient || !IS_TRANSIENT(win)));
     in_viewport = win_in_viewport(win);
     if (win->manager->usewinlist && DO_SKIP_WINDOW_LIST(win))
       in_viewport = 0;
+      in_viewport = win_in_viewport(win);
     if (win->button == NULL && in_viewport && is_state_selected) {
       insert_windows_button(win);
       if (win->manager->window_up == 0 && globals.got_window_list)
