--- fvwm-2.7.0.orig/configure.ac
+++ fvwm-2.7.0/configure.ac
@@ -100,7 +100,7 @@
 # installation paths
 transform=`echo "${program_transform_name}" | "$SED" -e 's/\\$\\$/\\$/'`
 PPACKAGE=`echo "${PACKAGE}" | "$SED" -e "${transform}"`
-FVWM_MODULESUBDIR=/${PPACKAGE}/${VERSION}
+FVWM_MODULESUBDIR=/${PPACKAGE}
 FVWM_DATASUBDIR=/${PPACKAGE}
 FVWM_DOCSUBDIR=/doc/${PPACKAGE}
 
@@ -111,8 +111,8 @@
   fi], [])
 
 
-FVWM_MODULEDIR='${libexecdir}'"$FVWM_MODULESUBDIR"
-FVWM_DATADIR='${datadir}'"$FVWM_DATASUBDIR"
+FVWM_MODULEDIR='${libdir}'"$FVWM_MODULESUBDIR"
+FVWM_DATADIR='${pkgdatadir}'
 FVWM_DOCDIR='${datadir}'"$FVWM_DOCSUBDIR"
 FVWM_PERLLIBDIR='${datadir}'"$FVWM_DATASUBDIR/perllib"
 FVWM_CONFDIR='${sysconfdir}'dnl used _only_ to search for system.fvwm2rc
@@ -1274,7 +1274,7 @@
 AC_SUBST(FVWMSCRIPT_DOMAIN)
 AC_SUBST(ALL_DOMAINS)
 
-LOCALEDIR="$FVWM_DATADIR/locale"
+LOCALEDIR='${datadir}'"/locale"
 with_gettext="yes"
 problem_gettext=""
 
