diff -urN fvwm-2.5.12.vanilla/configure.in fvwm-2.5.12/configure.in
--- fvwm-2.5.12.vanilla/configure.in	2004-10-06 02:42:02.000000000 +0200
+++ fvwm-2.5.12/configure.in	2005-02-21 21:46:00.825000040 +0100
@@ -72,7 +72,7 @@
 AM_CONFIG_HEADER(config.h)
 
 dnl installation paths
-FVWM_MODULESUBDIR=/${PACKAGE}/${VERSION}
+FVWM_MODULESUBDIR=/${PACKAGE}
 FVWM_DATASUBDIR=/${PACKAGE}
 
 AC_ARG_ENABLE(package-subdirs,
@@ -80,8 +80,8 @@
   [if test x"$enableval" = xno; then FVWM_MODULESUBDIR=""; FVWM_DATASUBDIR="";
   fi], [])
 
-FVWM_MODULEDIR='${libexecdir}'"$FVWM_MODULESUBDIR"
-FVWM_DATADIR='${datadir}'"$FVWM_DATASUBDIR"
+FVWM_MODULEDIR='${libdir}'"$FVWM_MODULESUBDIR"
+FVWM_DATADIR='${pkgdatadir}'
 FVWM_PERLLIBDIR='${datadir}'"$FVWM_DATASUBDIR/perllib"
 FVWM_CONFDIR='${sysconfdir}'dnl used _only_ to search for system.fvwm2rc
 
