diff -urN fvwm-2.5.12.vanilla/modules/FvwmPager/x_pager.c fvwm-2.5.12/modules/FvwmPager/x_pager.c
--- fvwm-2.5.12.vanilla/modules/FvwmPager/x_pager.c	2004-09-27 12:32:20.000000000 +0200
+++ fvwm-2.5.12/modules/FvwmPager/x_pager.c	2005-02-21 21:47:10.254445160 +0100
@@ -1078,6 +1078,29 @@
     else if((Event->xbutton.button == 1)||
 	    (Event->xbutton.button == 2))
     {
+		if( (Event->xbutton.button == 1) &&
+		  (Event->xbutton.subwindow != None) )
+		{
+		  PagerWindow *t;
+
+		  t = Start;
+		  while ((t != NULL)&&(t->PagerView != Event->xbutton.subwindow))
+		    t= t->next;
+
+		  if(t==NULL)
+		  {
+		    t = Start;
+		    while ((t != NULL)&&(t->IconView != Event->xbutton.subwindow))
+		      t= t->next;
+		  }
+
+		  if( t )
+		  {
+		    SendText(fd, "Silent Raise", t->w);
+		    SendText(fd, "Silent Focus", t->w);
+		  }
+		}
+
       for(i=0;i<ndesks;i++)
       {
 	if(Event->xany.window == Desks[i].w)
