diff -urp fvwm-2.5.16.orig/configure.ac fvwm-2.5.16/configure.ac
--- fvwm-2.5.16.orig/configure.ac	2006-01-20 10:17:13.000000000 +0100
+++ fvwm-2.5.16/configure.ac	2006-04-14 03:10:34.000000000 +0200
@@ -73,7 +73,7 @@ AM_INIT_AUTOMAKE(fvwm, ${version})
 AM_CONFIG_HEADER(config.h)
 
 # installation paths
-FVWM_MODULESUBDIR=/${PACKAGE}/${VERSION}
+FVWM_MODULESUBDIR=/${PACKAGE}
 FVWM_DATASUBDIR=/${PACKAGE}
 
 AC_ARG_ENABLE(package-subdirs,
@@ -81,8 +81,8 @@ AC_ARG_ENABLE(package-subdirs,
   [if test x"$enableval" = xno; then FVWM_MODULESUBDIR=""; FVWM_DATASUBDIR="";
   fi], [])
 
-FVWM_MODULEDIR='${libexecdir}'"$FVWM_MODULESUBDIR"
-FVWM_DATADIR='${datadir}'"$FVWM_DATASUBDIR"
+FVWM_MODULEDIR='${libdir}'"$FVWM_MODULESUBDIR"
+FVWM_DATADIR='${pkgdatadir}'
 FVWM_PERLLIBDIR='${datadir}'"$FVWM_DATASUBDIR/perllib"
 FVWM_CONFDIR='${sysconfdir}'dnl used _only_ to search for system.fvwm2rc
 
Only in fvwm-2.5.16: fvwm2.menu.m4
Only in fvwm-2.5.16: icons
Only in fvwm-2.5.16: system.fvwm2rc
