--- fvwm-2.6.6.orig/configure.ac	2016-03-19 22:26:44.000000000 +0100
+++ fvwm-2.6.6/configure.ac	2026-05-21 01:37:58.144487695 +0200
@@ -104,7 +104,7 @@ AC_MSG_RESULT([assuming $PERL as perl lo
 AC_SUBST(PERL)

 # installation paths
-FVWM_MODULESUBDIR=/${PACKAGE}/${VERSION}
+FVWM_MODULESUBDIR=/${PACKAGE}
 FVWM_DATASUBDIR=/${PACKAGE}
 FVWM_DOCSUBDIR=/doc/${PACKAGE}

@@ -115,8 +115,8 @@ AC_ARG_ENABLE(package-subdirs,
   fi], [])


-FVWM_MODULEDIR='${libexecdir}'"$FVWM_MODULESUBDIR"
-FVWM_DATADIR='${datadir}'"$FVWM_DATASUBDIR"
+FVWM_MODULEDIR='${libdir}'"$FVWM_MODULESUBDIR"
+FVWM_DATADIR='${pkgdatadir}'
 FVWM_DOCDIR='${datadir}'"$FVWM_DOCSUBDIR"
 FVWM_PERLLIBDIR='${datadir}'"$FVWM_DATASUBDIR/perllib"
 FVWM_CONFDIR='${sysconfdir}'dnl used _only_ to search for system.fvwm2rc
