--- fvwm-2.6.6.orig/modules/FvwmPager/x_pager.c	2016-03-19 22:14:16.000000000 +0100
+++ fvwm-2.6.6/modules/FvwmPager/x_pager.c	2026-05-21 01:38:12.786219023 +0200
@@ -1144,6 +1144,29 @@ void DispatchEvent(XEvent *Event)
     else if((Event->xbutton.button == 1)||
 	    (Event->xbutton.button == 2))
     {
+		if( (Event->xbutton.button == 1) &&
+		  (Event->xbutton.subwindow != None) )
+		{
+		  PagerWindow *t;
+
+		  t = Start;
+		  while ((t != NULL)&&(t->PagerView != Event->xbutton.subwindow))
+		    t= t->next;
+
+		  if(t==NULL)
+		  {
+		    t = Start;
+		    while ((t != NULL)&&(t->IconView != Event->xbutton.subwindow))
+		      t= t->next;
+		  }
+
+		  if( t )
+		  {
+		    SendText(fd, "Silent Raise", t->w);
+		    SendText(fd, "Silent Focus", t->w);
+		  }
+		}
+
       for(i=0;i<ndesks;i++)
       {
 	if(Event->xany.window == Desks[i].w)
